lib/install/examples/stimulus/setup.js

```diff
@@ -0,0 +1,8 @@
+
+
```

Comment:
Let's rename this file to `application.js` since that's the file it ends up in. And move this leading whitespace to the injection.